app/controllers/conference_rooms_controller.rb

```diff
@@ -0,0 +1,21 @@
+class ConferenceRoomsController < ApplicationController
+  include GoogleAuthentication
+  include GoogleEventClient
+
+  before_action :refresh_token
+  before_action :check_authentication
+
+  def show
+    create_props
+  end
+
+  private
+
+  def create_props
+    @props = {conference_room: conference_room}.compact
```

Comment:
Why compact? Does it make sense to have conference_room view without conference_view?